scss/bootstrap/overrides/_variables.scss

```diff
@@ -52,51 +49,34 @@ $gray-dark:                 $crm-copy !default;
 $gray-light:                $crm-background !default;
 $gray-lighter:              #F3F6F7 !default;
 $crm-gray-lightest:         #FAFAFB !default;
-$crm-black:                 #000000 !default;
 
 $brand-primary:             #0071BD !default;
-$brand-success:             #4D994D !default;
+$brand-success:             #44CB7E !default;
 $brand-info:                #0071BD !default;
-$brand-warning:             #BF5900 !default;
+$brand-warning:             #E6AB5E !default;
 $brand-danger:              #CF3458 !default;
 
+$crm-darken-percentage:     7% !default;
 
-$crm-help-success:          #d1e5ce;
-$crm-brand-primary-light:   #2574A9 !default;
-$crm-brand-primary-lighter: lighten(desaturate(adjust-hue(#3ab1c9, -10.0699), 36.9721), 48.2353) !default;
 
-$crm-color-neutral:         #949caa !default; // gray shade
 $crm-dark-blue:             #586277 !default;
 $crm-dark-grayblue:         #D3DEE2 !default;
 $crm-darker-grayblue:       #C2CFD8 !default;
-$crm-white:                 #FFF !default;
+$crm-white:                 #FFFFFF !default;
+$crm-black:                 #000000 !default;
+$crm-color-neutral:         $crm-darker-grayblue !default;
```

Comment:
if `$crm-darker-grayblue` is already defined, why do we need `color-neutral`?

Reply:
`$crm-color-neutral` is variable used in one of the component of datepicker.
The variable is going to get overriden in that component but, to prevent potential break in that until we decide where a component should be added. 
We need to have this color to allow other extensions to override it.
https://github.com/compucorp/civihr-employee-portal-theme/blob/staging/civihr_default_theme/scss/base/_variables.scss#L29